static/js/views/settings.js

```diff
@@ -1517,6 +1575,73 @@ const SettingsScreen = {
     return 0;
   },
 
+  showLocalizationSettings: function() {
+    this.localizationSettings.classList.remove('hidden');
+
+    const countrySelect = this.elements.localization.country;
+    countrySelect.innerHTML = '';
+    countrySelect.disabled = true;
+
+    API.getCountry().then((response) => {
```

Comment:
It would be nice to have a visible disabled state for the dropdown box (perhaps with `background-color: #48779a;`) if this comes back with an error or an empty response.